src/main/java/edu/kis/vh/nursery/stack/IntArrayStack.java

```diff
@@ -0,0 +1,45 @@
+package edu.kis.vh.nursery.stack;
+
+public class IntArrayStack implements StackInterface{
+
+	private static final int NUMBERS_CAPACITY = 12;
+
+	private int[] numbers = new int[NUMBERS_CAPACITY];
+
+	private int total = StackInterface.DEFAULT_EMPTY_STACK_VALUE;
+
+	@Override
+	public int getSize() {
+		return total;
+	}
+
+	@Override
+	public void push(final int in) {
+		if (!isFull())
+			numbers[total++] = in;
```

Comment:
Changing DEFAULT_EMPTY_STACK_VALUE value requires such fixes? It does not make sense. Incorrect fix.